Gaijinizer patch/Map081.txt

```diff
@@ -562,138 +610,156 @@ you.#MTLed
 
  気を確かにもってくれ。
 > CONTEXT: Map081/events/1/pages/2/77/Dialogue/yuuma
-Surely you touch the magic of the demon that will
-attack one after another now. My head is
-confused. Make sure you have your mind.#MTLed
+I'm sure it's just the magic of the demons
+attacking you that's confusing your mind.
+
+You'll be fine.#RonIffy
 > END STRING
 
 > BEGIN STRING
 一緒に……一緒にこの街から脱出するぞ。
 
 そうすれば……おまえも………
 > CONTEXT: Map081/events/1/pages/2/82/Dialogue/yuuma
-Together...I will escape from this city
-together. If you do so...you too......#MTLed
+Together... Let's escape this city together.
+
+If we can do that... you'll...#Ron
 > END STRING
 
 > BEGIN STRING
 分かった…………ユウマ。
 
 ………僕はもう………迷わないよ。
 > CONTEXT: Map081/events/1/pages/2/87/Dialogue/kazuya-3
-Okay ......... Yuuma....I'm not lost anymore
-... ....#MTLed
+Okay... Yuuma.
+
+...I know... what to do now...#Ron
 > END STRING
 
 > BEGIN STRING
 お前をこれ以上……
 
 待たせるわけにはいかないからな……
 > CONTEXT: Map081/events/1/pages/2/91/Dialogue/kazuya-3
-You are more than this ...... 
-I can not afford to wait ...#MTLed
+This is it for you...
+
+I can't keep him waiting...#Ron
 > END STRING
 
 > BEGIN STRING
 ………彼は、吹っ切れたようだね。
 > CONTEXT: Map081/events/1/pages/2/103/Dialogue/J_People1
-......... He seems to have blown off.#MTLed
+...He seems to have gone off the chain.#Ron
 > END STRING
```

Comment:
Maybe it's just a different set of expressions, but I think this would fit better:
>_**...He seems to have gone off the deep end.**_